boards/arm64/qemu_cortex_a53/qemu_arm64_kvm.yaml

```diff
@@ -0,0 +1,13 @@
+identifier: qemu_arm64_kvm
+name: QEMU Virt Emulation for AArch64 KVM
+type: qemu
+arch: arm64
+toolchain:
+  - zephyr
+  - cross-compile
+ram: 128
+testing:
+  default: true
```

Comment:
```suggestion
```
I see little point in making this board a default testing target when there already exists `qemu_cortex_a53` for the purpose of testing ARM64.